apps/namadillo/src/App/Common/TableWithPaginator.tsx

```diff
@@ -52,7 +52,7 @@ export const TableWithPaginator = <T,>({
         headProps={headProps}
       />
     );
-  }, [rows, tableProps, headProps]);
+  }, [headers, id, rows, tableProps, headProps]);
```

Comment:
Thinking loud, maybe we could enable the lint `react-hooks/exhaustive-deps` as warning for this. But we probably need to accept warnings on the build, otherwise it'll be a big refactoring